nl_server/routes.py

```diff
@@ -47,13 +48,12 @@ def healthz():
   if default_embeddings_loaded:
     return 'OK', 200
 
-  default_index_type = current_app.config[
-      config.EMBEDDINGS_SPEC_KEY].default_index
+  default_index_type = current_app.config[config.ENV_KEY].default_indexes[0]
```

Comment:
[0] may not be safe?  LHS has a check before access.

Reply:
good point, updated the check.